### PR TITLE
Merge 3.x into master

### PR DESCRIPTION
**Pull request by dmaicher**

### CODE REVIEW

**Review by franmomu — APPROVED** [review]

File: src/Resources/config/doctrine_mongodb.php [hunk 1]
```diff
@@ -39,6 +39,7 @@
         ->set('sonata.admin.builder.doctrine_mongodb_form', FormContractor::class)
             ->args([
                 (new ReferenceConfigurator('form.factory'))->ignoreOnInvalid(),
+                (new ReferenceConfigurator('form.registry'))->ignoreOnInvalid(),
```
Comment (franmomu) [inline src/Resources/config/doctrine_mongodb.php:42]:
```suggestion
                (new ReferenceConfigurator('form.registry')),
```
I think it's mandatory, probably `form.factory` too.

Comment (franmomu) [reply src/Resources/config/doctrine_mongodb.php:42]: nevermind, it could be done in another PR, maybe there are more like this.

Comment (dmaicher) [reply src/Resources/config/doctrine_mongodb.php:42]: indeed. I will fix it on 3.x as well.

Comment (dmaicher) [reply src/Resources/config/doctrine_mongodb.php:42]: Yeah there are quite a few `ignoreOnInvalid()` :confused:

**Pull request merged**